Dockerfile.slim

```diff
@@ -0,0 +1,9 @@
+FROM alpine:3.11
+MAINTAINER sig-platform@spinnaker.io
+RUN apk --no-cache add --update bash openjdk11-jre
+RUN addgroup -S -g 10111 spinnaker
+RUN adduser -D -S spinnaker -u 10111 -G spinnaker
+COPY keel-web/build/install/keel /opt/keel
```

Comment:
I get a `keel-web` dir, not `keel`, from running `keel-web:installDist`. Does Docker build this OK for you?

Reply:
sorry I should have been clearer in my last comment.

I only changed `keel-web` to `keel` in the last commit of this PR. reason is, once #1410 is merged, the `io.spinnaker.package` plugin will create the install script as `keel` and not `keel-web` anymore. I didnt want to have to fix up references in the dockerfiles after $1410 is merged, hence [this comment](https://github.com/spinnaker/keel/pull/1400#issuecomment-670227267) on this PR depending on #1410.

I guess merging this line of work should go as #1409, #1410, and then #1400.

Reply:
Got it. The Dockerfiles look good to me, and I merged #1410 before seeing this comment, so I guess I'll watch for kork changes so that we can get this merged. I didn't see a PR linked in the issue -- were you planning to make that change in kork as well?

Reply:
Actually, I don't think there's any harm in getting this merged before #1409 is fixed. Let me know.

Reply:
there isn't any harm except that it wont build properly in the meantime. so if there is no dependency to it, then we are good.

Reply:
> I didn't see a PR linked in the issue -- were you planning to make that change in kork as well?

I can PR the kork change too but then a new release needs to be cut for it and keel will have to bump the kork version. I think it is easier if as part of #1409, someone with permissions to do all three tasks above can do all of this in one go.